tests/test_nautilus_librarian/utils.py

```diff
@@ -0,0 +1,41 @@
+import pathlib
+import subprocess  # nosec
+
+
+def debug_execute_console_command(multiline_command):
+    return execute_console_command(
+        multiline_command, print_output=True, print_command=True
+    )
+
+
+def execute_console_command(multiline_command, print_output=False, print_command=False):
+
+    commands = multiline_command.splitlines()
+
+    full_output = ""
+
+    for command in commands:
+
+        if print_command:
+            print(command.strip())
+
+        process = subprocess.run(
+            command,
+            shell=True,  # nosec
+            check=True,
+            stdout=subprocess.PIPE,
+            universal_newlines=True,
+        )
+
+        output = process.stdout
+
+        if print_output:
+            print(output)
+
+        full_output += output
+
+    return full_output
+
+
+def get_fixtures_dir():
```

Comment:
What is the purpose of this function?

Reply:
It's a helper function to use only in tests. It gives you the test folder where I'm going to store tests fixtures like: example images, gpg keys, etcetera. I can add this description :-) to the docstring.